docs/sources/getting-started/getting-started-elasticsearch.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "With Grafana and Elasticsearch"
+description = "Guide for getting started with Grafana"
+keywords = ["grafana", "intro", "guide", "started", "Elastic", "Elasticsearch"]
+weight = 400
++++
+
+# Getting started with Grafana and Elasticsearch
+
+Elasticsearch is a popular open source search and analytics engine for which Grafana provides out-of-the-box support. This topic walks you through the steps to create your first Elasticsearch backed dashboard in grafana to display any kind of data stored in your Elasticsearch database.
+
+> **Note:** Before you begin, use the instructions in [Getting started with Grafana]({{< relref "getting-started.md" >}}) to install Grafana and build your first dashboard.
+
+## Step 1. Download and install Elasticsearch
+
+Elasticsearch can be installed on many different operating systems. Refer to the [Installing Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html) for a complete list of all available options.
+
+Alternately you can install Elasticsearch using the resources available in [grafana/grafana](https://github.com/grafana/grafana) GitHub repository. Here you will find a collection of supported data sources, including Elasticsearch, along with test data and preconfigured dashboards for use.
+
+1. Clone the [grafana/grafana](https://github.com/grafana/grafana/tree/master) repository to your local system.
+1. Install Docker or verify that it is installed on your machine.
+
+1. Within your local `grafana` repository, change directory to [devenv](https://github.com/grafana/grafana/tree/master/devenv).
+
+   ```
+   cd devenv
+   ```
+
+1. Run the bash command to setup data sources and dashboards.
+
+   ```
+   ./setup.sh
+   ```
+
+1. Restart the Grafana server.
+1. Change directory back to the root directory.
+
+   ```
+   cd ..
+   ```
+
+1. Run the make command to create and start Elasticsearch.
+
+   ```
+   make devenv sources=elastic7
+   ```
+
+This creates and starts Elasticsearch, Filebeat, Metricbeat, Kibana, and a process that sends random data to your new Elasticsearch instance.
+
+> **Note**: Filebeat, Metricbeat, and Kibana are outside the scope of this topic.
+
+## Step 2. Configure and add the Elasticsearch data source
+
+Once you have your Elasticsearch instance up and running and some process pushing data to it, you can configure your data source within Grafana.
+
+For information on how to add a datasource, refer to [Add a data source](/docs/grafana/latest/datasources/add-a-data-source/).
+See also, [Using Elasticsearch in Grafana](/docs/grafana/latest/datasources/elasticsearch/) for detailed list of all available configuration options for Elasticsearch.
```

Comment:
```suggestion
Refer to [Using Elasticsearch in Grafana](/docs/grafana/latest/datasources/elasticsearch/) for detailed list of all available configuration options for Elasticsearch.
```

Reply:
Is this topic title still correct, or did we change it to Elasticsearch data source? The link text should match the topic title whenever possible.

Reply:
Not sure i got this. "Using Elasticsearch in Grafana" is the title of the page this links to